ocean/nemo/nemolite2d/manual_versions/psykal_dm/nemolite2d.f90

```diff
@@ -120,15 +109,16 @@ program gocean2d
 
   !! finalise the model run
   call model_finalise()
-  !call likwid_markerClose()
-
+  
   call model_write_log("((A))", 'Simulation finished!!')
 
+  call gocean_finalise()
+
 end program gocean2d
 
 !+++++++++++++++++++++++++++++++++++
 
-subroutine step(istp,           &
+subroutine step(grid, istp, &
```

Comment:
Remove commented invoke at the end of this subroutine

Reply:
Done.